src/SUMMARY.md

```diff
@@ -38,6 +38,10 @@
     - [Supporting Algorithms](./threshold-ecdsa/threshold-ecdsa-construction/actual-construction/supporting-algorithms.md)
     - [Identifying Aborts](./threshold-ecdsa/threshold-ecdsa-construction/actual-construction/abort-identification.md)
   - [Implementation Guideline](./threshold-ecdsa/intended-implementation/our-intended-implementation.md)
+    - [Threshold ECDSA from Secp256k1](./threshold-ecdsa/intended-implementation/threshold-ecdsa-from-secp256k1.md)
+    - [Threshold ECDSA from Ed25519](./threshold-ecdsa/intended-implementation/threshold-ecdsa-from-ed25519.md)
+    - [Threshold ECDSA from Sr25519](./threshold-ecdsa/intended-implementation/threshold-ecdsa-from-sr-25519.md)
```

Comment:
`Ed25519` and `Sr25519` should be EdDSA?

Reply:
they are both Schnorr signatures